vendor/src/github.com/dancannon/gorethink/query.go

```diff
@@ -0,0 +1,290 @@
+package gorethink
```

Comment:
This would be an example of something that we wouldn't version. We can let gb handle pulling down all of the dependencies for the developer.